encryption/rollo/rollo-II/bench/avx_rollo-II-128_opt.yaml

```diff
@@ -0,0 +1,6 @@
+platform: AVX2
```

Comment:
this should be processor (speed), compiler, etc.

Reply:
please see https://github.com/cryptoeng/pqdb/pull/124#discussion_r456401690 about the filenames